lib/requester/util.js

```diff
@@ -116,15 +130,23 @@ module.exports = {
      *
      * @todo: Move this to the SDK.
      * @param request
+     * @param options
      */
-    getRequestBody: function (request) {
-        var options,
-            mode = _.get(request, 'body.mode'),
+    getRequestBody: function (request, options) {
+        var mode = _.get(request, 'body.mode'),
             body = _.get(request, 'body'),
-            empty = request.body && request.body.isEmpty && request.body.isEmpty(),
-            content;
+            empty = body ? body.isEmpty() : true,
```

Comment:
Who will check function type of body? It was there previously. God knows what edge cases will break because of this. 

`_.isFunction(body && body.isEmpty) ? body.isEmpty() : true;`